tests/compute/nonuniformres-as-function-parameter.slang

```diff
@@ -0,0 +1,95 @@
+//TEST:SIMPLE(filecheck=CHECK-SPV):-target spirv -entry main -stage compute
+//TEST:SIMPLE(filecheck=CHECK-GLSL-SPV):-target spirv -entry main -stage compute -emit-spirv-via-glsl
+//TEST:SIMPLE(filecheck=CHECK-GLSL):-target glsl -entry main -stage compute
+//TEST:SIMPLE(filecheck=CHECK-HLSL):-target hlsl -entry main -stage compute
```

Comment:
Please avoid using minus character(-) as a part of the user-defined keyword.
It is legal to do it but it can cause a confusion, because the words after minus characters are supposed to be a suffix for predefined commands like -NOT or -NEXT.
I suggest to rename them to CHECK_SPV with an underscore character.